### DISCUSSION TITLE
Installation Help: Importing Data

### DISCUSSION THREAD

**Discussion by anubhav06** [osm-search/Nominatim#2636] [Q&A]
Hey. I was following the docs for [importing the data](https://nominatim.org/release-docs/develop/admin/Import/#initial-import-of-the-data)

![1](https://user-images.githubusercontent.com/43821031/158814161-cb0de644-fdea-4991-9ab6-fada23c3c5ca.png)

When I run the installation command `nominatim import --osm-file <data file> 2>&1 | tee setup.log`, as shown in the docs, I get the following error:

![0](https://user-images.githubusercontent.com/43821031/158814312-61638707-e896-4338-9adc-83d2488d46fe.png)

Here are the roles available in postgresql
![2](https://user-images.githubusercontent.com/43821031/158816201-ea47d3ba-3e81-4c0b-8b2f-83ee9281e4c6.png)

Any help is appreciated. Thanks!
(Also, I am a beginner in Linux so I'm sorry if this is a simple question)

**Comment by anubhav06** [comment]
For anyone looking at this in the future, I found a solution as below:

Create a new role for which it says does not exist (in my case anubhav) using ```sudo -u postgres createuser owning_user```

**Reply by mtmail** [reply]
You might need to add ‘-s’ as well. https://nominatim.org/release-docs/latest/appendix/Install-on-Ubuntu-20/#setting-up-postgresql has a section on user account creation.
